app/views/idv/doc_auth/upload.html.erb

```diff
@@ -89,7 +92,16 @@
       </button>
     <% end %>
   </div>
+  <div id="upload-comp-liveness"
+       class='sm-col sm-col-12 <%= 'hidden' unless liveness_checking_enabled? %>'>
+    <strong><%= t('doc_auth.info.upload_from_computer') %></strong>
+    <%= t('doc_auth.info.upload_from_computer_not_allowed') %>
+  </div>
 </div>
 
 <%= render 'idv/doc_auth/start_over_or_cancel' %>
 <%= javascript_pack_tag 'image-preview' %>
+
+<% unless Figaro.env.document_capture_react_enabled == 'false' %>
```

Comment:
With the `unless` and the `false` this is kind of a double negative... WDYT about flipping it?

```erb
<% if Figaro.env.document_capture_react_enabled == 'true' %>
```

Reply:
Yeah, I noticed after I committed. I just stole that from something Doug put up recently. It's weird and I'll flip it.

Reply:
IIRC, originally it wasn't included in `application.yml.default`, as it was intended for local override. At that time, `nil` and `'true'` were used interchangeably.

See: #3994

That being said, it might also be time to retire this configuration? I think @solipet expected it to be temporary for testing the fallback flow. We can still test this flow by disabling JavaScript (i.e. through Chrome DevTools).

Reply:
That is also an option. If it was only meant to be temporary I can remove the check. I have been using the browser based "disable javascript" to test anyway.

Reply:
I just removed the check because it wasn't really relevant anymore.